src/models/model.cpp

```diff
@@ -364,6 +364,11 @@ void Model::CreateSessionOptionsFromConfig(const Config::SessionOptions& config_
     session_options.SetEpContextFilePath(config_session_options.ep_context_file_path.value().c_str());
   }
 
+  if (config_session_options.provider_options.empty() && config_session_options.use_env_allocators) {
```

Comment:
is it also possible to specify CPU EP provider options? would `provider_options` not be empty in that case?

Reply:
The only way to set CPU provider is by leaving the provider options empty. We do not have a way to explicitly set the CPU provider right now.